style.css

```diff
@@ -419,14 +419,177 @@ input:focus {
   border-color: rgb(98, 108, 201);
 }
 
+.headerlish {
+  display: none;
+}
+
 @media (min-width: 375px) {
   .content-section {
     background-size: cover;
   }
 }
 
+/* desktop version */
+
 @media (min-width: 768px) {
+  .icon-Union {
+    display: none;
+  }
+
+  .topBar-nav {
+    position: sticky;
+    top: 0;
+    background-color: white;
+  }
+
+  .headerlist {
+    display: flex;
+    gap: 8px;
+    list-style-type: none;
+    padding-right: 20px;
+    font-family: var(--primary-font);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    position: sticky;
+    top: 0;
+  }
+
+  .headerlist a:hover {
+    color: rgb(24, 155, 160);
+  }
+
   .content-section {
+    background-image: url(img/Header\ bg-desktop.png);
+    background-size: contain;
+    display: flex;
+    height: 718px;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .main-Container {
+    width: 60%;
+    margin-left: 0;
+    padding-top: 0;
+  }
+
+  .work-container {
+    display: flex;
+    margin: 30px 10px;
+  }
+
+  .cards {
+    width: auto;
+    flex-direction: row;
+    gap: 25px;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .snapshot-portofolio img {
+    padding-bottom: 0;
+  }
+
+  .blocks {
+    gap: 10px;
+  }
+
+  .tframes {
+    gap: 12px;
+  }
+
+  .tframes h3 {
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .frames-list {
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .reverse-block {
+    flex-direction: row-reverse;
+    gap: 20px;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .project-text {
+    align-items: center;
+    font-size: 16px;
+    width: 100%;
+  }
+
+  .about-section {
+    display: flex;
+    height: auto;
+    gap: 12px;
+    flex-direction: row;
     width: 80%;
+    justify-content: center;
+    margin: 0;
+  }
+
+  .aboutWrapper {
+    padding-top: 10px;
+    width: 40%;
+  }
+
+  .about-Container .headingTxt {
+    width: 100%;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .about-Container .text-2 {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .skil-List {
+    width: 50%;
+    padding-top: 0;
+  }
+
+  .skil-List-List {
+    flex-direction: row;
+  }
+
+  .skil-List-List-values {
+    flex-direction: column;
+    align-items: flex-start;
+    width: 120px;
+    height: auto;
+  }
+
+  /* comtact me section */
+
+  .contact-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-image: url(img/contact-desktop-bg.png);
+    background-size: contain;
+    gap: 15px;
+    background-position: center;
+  }
+
+  .contact-section {
+    width: 50%;
+  }
+
+  .contact-form-container {
+    width: 65%;
   }
```

Comment:
- [ ] Great work on the desktop version of the Contact Me section! :+1: However, to align it with the Figma template, I recommend centering the form and the "Get in touch" button. This will ensure that the section looks consistent and visually appealing.

**Your design**

![Screenshot from 2023-04-01 16-22-29](https://user-images.githubusercontent.com/105117832/229314471-ec254cdc-207f-4083-9b0e-50da9f1b2cd7.png)

**Figma**

![Screenshot from 2023-04-01 16-22-46](https://user-images.githubusercontent.com/105117832/229314476-e3052840-a2e9-4c40-9149-867d193f672a.png)